project/MiMa.scala

```diff
@@ -46,7 +46,7 @@ object MiMa extends AutoPlugin {
             else Seq.empty
           } ++ akka24StreamVersions ++ akka24WithScala212 ++ akka25Versions
 
-        case "2.12" =>
+        case _ =>
```

Comment:
Perhaps rename `akka24WithScala212` to `akka24WithAtLeastScala212`?

Reply:
I have incorporated these changes in PR https://github.com/akka/akka/pull/23620